storage_service/locations/signals.py

```diff
@@ -44,4 +47,26 @@ def report_failed_fixity_check(sender, **kwargs):
 Full failure report (in JSON format):
 {}
 """.format(kwargs["uuid"], kwargs["location"], kwargs["report"])
+    try:
+        package = models.Package.objects.get(uuid=kwargs['uuid'])
+
+        log = models.FixityLog()
+        log.package = package
+        report_data = json.loads(kwargs['report'])
+        log.error_details = report_data['message']
+        log.save()
+
+        package.last_fixity_check_result = False
+        package.last_fixity_check_date = log.time_reported
+        package.save()
+    except:
+        LOGGER.exception("Error logging fixity failure.")
```

Comment:
Is this an ERROR level log or something lesser?  Tracebacks can be added to any log message with [exc_info](https://docs.python.org/2/library/logging.html#logging.debug)